src/ProjectService.Data/TaskRepository.cs

```diff
@@ -48,18 +49,44 @@ public TaskRepository(IDataProvider provider)
             _provider = provider;
         }
 
+        public Guid CreateTask(DbTask newTask)
+        {
+            _provider.Tasks.Add(newTask);
+            _provider.Save();
+
+            return newTask.Id;
+        }
+
+        public bool IsExist(Guid id)
+        {
+            return _provider.Tasks.FirstOrDefault(x => x.ParentId == id) != null;
+        }
```

Comment:
Why not `return _provider.Tasks.FirstOrDefault(x => x.Id == id) != null;`

Reply:
because Id is task parameter, ParendId it's Id another task